src/main/scala/uk/gov/hmrc/sbtdistributables/SbtDistributablesPlugin.scala

```diff
@@ -37,6 +37,7 @@ object SbtDistributablesPlugin extends AutoPlugin {
   val publishTgz = TaskKey[sbt.File]("publish-tgz", "publish tgz artifact")
 
   lazy val extraFiles = SettingKey[Seq[File]]("extraFiles", "Extra files to be added to the tgz")
+  lazy val executableFilesInTar = SettingKey[Seq[File]]("executableFilesInTar", "Files to made executable in tar")
```

Comment:
`executableFilesInTar` should have a default value of empty sequence as most teams don't need